style.css

```diff
@@ -256,570 +421,461 @@ body {
+    border: 1.5px solid #000;
+    padding: 11px 22px;
+    margin-right: 9px;
+    margin-left: 53px;
   }
 
-  .header__menu-item {
-    height: 4px;
-    width: 32.7px;
-    margin-bottom: 7px;
+  .second__add {
+    margin-left: 0;
+    border: 1.5px solid #000;
+    padding: 11px 63px;
   }
 
-  .header__logo p {
-    margin: 11px 4px;
-    color: black;
-    font-size: 33px;
-    letter-spacing: 2px;
+  .second__other {
+    font-weight: 500;
+    font-size: 13px;
+    line-height: 1.2;
+    text-decoration: none;
+    color: #000;
+    margin-left: 5px;
+    margin-right: 10px;
+    margin-top: 84px;
+    display: inline-block;
   }
+}
 
-  .text__container {
-    display: flex;
-    align-items: flex-end;
-    width: 100%;
-    height: 462px;
-    letter-spacing: 5px;
+@media (min-width: 1440px) {
+  .second {
+    padding-top: 167px;
+    grid-column-gap: 74px;
   }
 
-  .text__title {
-    position: inherit;
-    align-items: flex-end;
-    font-size: 55px;
-    line-height: 57px;
-    padding-left: 36px;
-    letter-spacing: 1px;
+  .second__container {
+    height: 437px;
   }
 
-  .vectormob {
-    position: absolute;
-    width: 43px;
-    height: 23px;
-    left: 45.5%;
-    top: 511px;
-    display: block;
+  .second__add {
+    padding: 11px 63px;
   }
-}
 
-@media (max-width: 360px) {
-  .first-container {
-    padding: 22px 0;
-    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
-    background-size: 360px;
-    background-position: center;
+  .second__bag-dsk {
+    width: 739px;
+    margin-top: 57px;
+    margin-left: 23px;
   }
 
-  .header__inner {
-    width: 100%;
-    height: 65px;
-    background: #fff;
-    z-index: 2;
+  .second__quantity {
+    margin-right: 15px;
+    padding: 11px 22px;
   }
 
-  .header__menu {
-    background: #ededed;
-    z-index: 3;
-    height: 65px;
-    width: 65px;
+  .second__other {
+    margin-top: 84px;
   }
 
-  .header__menu-item {
-    height: 4px;
-    width: 27.7px;
-    margin-bottom: 7px;
+  .second__quantity::after {
+    content: '+';
+    margin-left: 19px;
   }
 
-  .header__logo {
-    margin: 4px 28px;
+  .second__quantity::before {
+    content: '-';
+    margin-right: 19px;
   }
 
-  .header__logo p {
-    margin: 10px 0;
-    color: black;
-    font-size: 28px;
-    letter-spacing: 2px;
+  .third__line {
+    margin-left: 389px;
   }
 
-  .text__container {
-    display: flex;
-    align-items: flex-end;
-    width: 100%;
-    height: 381px;
-    letter-spacing: 5px;
+  .third__img-dsk {
+    left: 89px;
   }
 
-  .text__title {
-    position: inherit;
-    align-items: flex-end;
-    font-size: 46px;
-    line-height: 49px;
-    padding-left: 28px;
-    letter-spacing: 2px;
+  .third__block {
+    margin-right: 149px;
   }
 
-  .vectormob {
-    position: absolute;
-    width: 38px;
-    height: 21px;
-    left: 45%;
-    top: 426px;
-    display: block;
+  .third__next {
+    margin-right: 81px;
+    margin-top: 136px;
   }
-}
-
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)